src/main/scala/is/hail/variant/Variant.scala

```diff
@@ -54,7 +54,7 @@ object CopyState extends Enumeration {
 object AltAllele {
 
   def fromRegionValue(m: MemoryBuffer, offset: Long): AltAllele = {
-    val t = TAltAllele.representation
+    val t = TAltAllele().representation
```

Comment:
So this should be `fundamentalType`, I think.

Reply:
actually, I think these are probably okay as `representation` because all the `fundamentalType`s are used at a lower level than these calls are made and the structure of the region value calls mimics the representation-level structure. But I don't have super strong feelings and I think fundamentalType would also work